### PR TITLE
test: test gen and build workflow

### PR DESCRIPTION
**Pull request by vince-fugnitto**
Tests #98.

### CODE REVIEW

File: templates/hello-world/contribution.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const <%= params.extensionPrefix %>Command = {
 };
 
 @injectable()
-export class <%= params.extensionPrefix %>CommandContribution implements CommandContribution {
+export class <%= params.extensionPrefix %>CommandContribution implements CommandContribution
```
Comment (JonasHelming) [inline templates/hello-world/contribution.ts:11]: I think there is a mistake in this line :-)

**Review by JonasHelming — CHANGES_REQUESTED** [review]

**Comment by vince-fugnitto** [comment]
@JonasHelming it looks like the workflow fails fast (meaning it fails as soon as 1 template fails).
I think it'd be better to continue so we can generate all templates and build to identify all issues.

File: templates/hello-world/contribution.ts (hunk 1 above)
Comment (vince-fugnitto) [reply templates/hello-world/contribution.ts:11]: Ahh you're right ;)

**Comment by vince-fugnitto** [comment]
Closing since #98 is merged :+1:

**Pull request closed without merging**